package_control/package_manager.py

```diff
@@ -1415,6 +1416,7 @@ def add_extracted_dirs(dir_):
             # if there is an error removing a file. The next time there is an
             # upgrade, it should be cleaned out successfully then.
             clear_directory(package_dir, extracted_paths)
+            clear_directory(package_dir, set(extracted_paths))
```

Comment:
What's the intention of calling `clear_directory` twice for `extracted_paths`?

Reply:
Looks like I just screwed up line staging :)